### PR TITLE
Add content for Fortran language

### PR DESCRIPTION
**Pull request by jakafle**
Getting started with Fortran in Fedora.

### CODE REVIEW

**Comment by pvalena** [comment]
Hello!

Thanks for the PR.

Please also create corresponding PR (if there's not one already) with the logo in `website`  repository:

https://github.com/developer-portal/website/tree/master/static/logo

File: tech/languages/fortran/fortran-installation.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+```
+gfortran your_source.f
+```
+
+This will produce a binary file created from your source code. By default, this binary
+file is named `a.out`.
+You can run this binary by typing:
+
+```
+$ ./a.out
```
Comment (pvalena) [inline tech/languages/fortran/fortran-installation.md:31]: I think this should be in sync with the sample `your_source.f` file.

File: tech/languages/fortran/fortran-installation.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+```
+$ sudo dnf install gcc-gfortran
+```
+
+This command should download and install package with compiler called gfortran. To compile
+and link a program written in Fortran, simply run the `gfortran` command like this:
+
+```
+gfortran your_source.f
```
Comment (pvalena) [inline tech/languages/fortran/fortran-installation.md:23]: The line should start with `$ `, as per:

https://developer-portal.github.io/contributing/write-general-writing-hints.html

File: tech/languages/fortran/fortran-installation.md (hunk 1 above)
Comment (jakafle) [reply tech/languages/fortran/fortran-installation.md:30]: Hello, I have added the compiler option to specify the output file name.

File: tech/languages/fortran/fortran-installation.md (hunk 2 above)
Comment (jakafle) [reply tech/languages/fortran/fortran-installation.md]: Hello, this issue has been resolved.

**Comment by pvalena** [comment]
LGTM, thanks!

**Pull request merged**